### PR TITLE
Principal propagation

### PR DESCRIPTION
**Pull request by tomfrenken**
## What Has Changed?

This PR adds documentation for principal propagation in K8s environments.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/js/guides/migrate-sdk-application-from-btp-cf-to-kubernetes.mdx [hunk 1]
```diff
@@ -732,7 +733,7 @@ kubectl create secret docker-registry <your-registry-secret> \
   --docker-server=deploy-releases.docker.repositories.sap.ondemand.com
 ```
 
-6. Create a `values.yml` containing the configuration that suits your desired operational mode, for the available operational modes refer to the [documentation](https://wiki.wdf.sap.corp/wiki/display/NGP/Operational+modes#Operationalmodes-Singletenantinatrustedenvironment).
+6. Create a `values.yaml` file containing the configuration that suits your desired operational mode of the connectivity proxy, for the available operational modes refer to the [documentation](https://wiki.wdf.sap.corp/wiki/display/NGP/Operational+modes#Operationalmodes-Singletenantinatrustedenvironment).
```
[the remark this answers is not in the document]
Comment (tomfrenken) [reply docs/js/guides/migrate-sdk-application-from-btp-cf-to-kubernetes.mdx:736]: They've now made the documentation public, I've updated the links

File: docs/js/guides/migrate-sdk-application-from-btp-cf-to-kubernetes.mdx [hunk 2]
```diff
@@ -770,20 +771,207 @@ config:
...
+url: aHR0cHM6Ly9teS1hcGkuYXV0aGVudGljYXRpb24uc2FwLmhhbmEub25kZW1hbmQuY29tCg==
 ```
 
 8. Finally, add the binding to your `deployment.yml`, the same way you would add any other binding.
+
+## Principal Propagation
+
+Similar to the SAP BTP CloudFoundry way of propagating a principal to handle multi-tenancy, you can also use the Application Router (AppRouter) in Kubernetes.
+
+1. You need an XSUAA instance that can redirect to a Kubernetes URI, for this the parameter `redirect-uris` is important, however if you use one parameter while creating a service, you have to use all of them.
```
[the remark this answers is not in the document]
Comment (tomfrenken) [reply docs/js/guides/migrate-sdk-application-from-btp-cf-to-kubernetes.mdx:798]: When we create a service, we can use parameters to specify some field that deviates from the default, like in this case we only want to change the value of `redirect-uris`. However, we can't just change one value; instead, we always have to override the entire template, even if we only want to change one value.

**Comment by tomfrenken** [comment]
@artemkovalyov updated now

**Review by artemkovalyov — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**